src/controllers/organisation/model/organisationalUnitFactory.ts

```diff
@@ -0,0 +1,21 @@
+import {OrganisationalUnit} from './organisationalUnit'
+
+export class OrganisationalUnitFactory {
+	constructor() {
+		this.create = this.create.bind(this)
+	}
+
+	public create(data: any): any {
+		const organisationalUnit: OrganisationalUnit = new OrganisationalUnit()
+
+		organisationalUnit.id = data.id
+		organisationalUnit.name = data.name
+		organisationalUnit.code = data.code
+		organisationalUnit.paymentMethods = data.paymentMethods
+		organisationalUnit.subOrgs = data.subOrgs
```

Comment:
Shouldn't the sub organisationalUnits be passed to create as well
`organisationalUnit.subOrgs = (data.subOrgs || []).map(this.create)`